### DISCUSSION TITLE
Tab completion when in command mode?

### DISCUSSION THREAD

**Discussion by raisinbl** [JetBrains/ideavim#872] [Q&A]
I just using ideavim recently and found that, when I'm in command mode (`: <command>`) then when I hit tab, it will go back to normal mode; tab again, it go back to command mode. Seems like I need tab btn to completion on command mode. How can I do that?

**Comment by AlexPl292** [comment]
Hi, I think this is one known issue [VIM-258](https://youtrack.jetbrains.com/issue/VIM-258)

**Reply by raisinbl** [reply]
seems like this feature still not implemented after 11y+ since the first raised issue.
